CHANGES.rst

```diff
@@ -102,7 +102,7 @@ tweakreg
 
 - Output source catalog file now respects ``output_dir`` parameter. [#8386]
 
-- Improved how a image group name is determined. [#8426]
```

Comment:
Why is the entry for #8426 being removed? Was that accidental?

Reply:
Thanks for the catch. This entry was lost in one of the updates to this PR. I expect this PR will need to be updated again once #8450 is merged. Would you prefer that I mark this as draft until that point or leave it open?